props/saber_fett263_buttons.h

```diff
@@ -257,7 +326,102 @@ SaberFett263Buttons() : PropBase() {}
       case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_OFF | BUTTON_AUX):
         previous_preset();
         return true;
+
+       // Gesture Controls
+		
+       // Battle Mode
+		    
+#ifdef FETT263_BATTLE_MODE
+       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON | BUTTON_AUX):
+         if (!battle_mode_) {
+           battle_mode_ = true;
+           hybrid_font.SB_Force();
```

Comment:
This is weird. Why play the Force sound when battle mode is enabled/disabled?

Reply:
It was to let user know it toggled but wanted it to be something that would make some "in universe" sense, like using the force to prepare for battle or recover from battle.

Reply:
but how do you know if it's turning on or off?

Reply:
good point, exiting will play Exit Color Change sound instead